util/types/src/core/views.rs

```diff
@@ -298,6 +298,15 @@ macro_rules! define_inner_getter {
 }
 
 impl TransactionView {
+    /// Destruct TransactionView to get wtx_hash
+    pub fn destruct_wtx_hash(self) -> packed::Byte32 {
+        let TransactionView {
+            witness_hash: wtx_hash,
+            ..
+        } = self;
+        wtx_hash
```

Comment:
Not sure but `self.witness_hash` should just work

```suggestion
        self.witness_hash
```

Reply:
Not sure too. @quake

Reply:
We have confirmed that holding a witness_hash will not prevent the release of TransactionView.